content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,22 @@
+# Getting Started with Entity Framework Core
+
+## Who is this tutorial meant for?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
+
+If you know what you're looking for, feel free to browse by topic on the side. If not, welcome! Let's get started together!
```

Comment:
"If you know what you're looking for, feel free to browse by topic on the side." => I get this, but I think the use of sidebar navigation is commonplace enough that we don't need to state they can browse by topic. It ends up just being more for the user to read.

"If not, welcome! Let's get started together!" => I understand the appeal to saying this, but it doesn't really match the tone of the other tutorials. I'd like the tone to be less like an individual person's blog. That might just be my perception, though.